pom.xml

```diff
@@ -237,6 +237,15 @@
     <error-prone.version>2.24.0</error-prone.version>
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
+    <error-prone.configuration-args>
+      <!-- Disabled rules as we do not see value in them. -->
+      -Xep:JUnitClassModifiers:OFF
+      -Xep:JUnitMethodDeclaration:OFF
+      -Xep:JUnitValueSource:OFF
```

Comment:
Why not activate this pattern?

Reply:
https://error-prone.picnic.tech/bugpatterns/JUnitValueSource/

We do not use this pattern of test design, we eliminating all tests that are unit tests, we do all tests by full checkstyle execution.

Reply:
https://github.com/checkstyle/checkstyle/wiki/Checkstyle-GSoC-2021-Project-Ideas#project-name-adaptation-of-behavior-driven-development-ideas-for-testing-of-source-code-validation-algorithms

Reply:
For now I added: `<!-- We prefer different style of writing tests. -->`. 

Let me know if you prefer something else.

A quick search showed me that in the project there are only 2 parameterized tests, both use `@ValueSource`.